config/initializers/decidim.rb

```diff
@@ -4,9 +4,12 @@
   config.application_name = "Seattle participatory budgeting"
   config.mailer_sender = Rails.application.config.mailer_sender
 
-  # Uncomment this lines to set your preferred locales
   config.default_locale = :en
-  config.available_locales = [:en, "zh-Hant", :ko, "zh-Hans", :so, :es, :tl, :vi, :fi, :sv]
+
+  # n.b. we are non-standard using locale names `zhHant` and `zhHans` to avoid
+  # hyphens, because of a bug in the plans module. Details were added to the
+  # README.
+  config. = [:en, :zhHant, :ko, :zhHans, :so, :es, :tl, :vi, :fi, :sv]
```

Comment:
We will need to update the locales for the organization when this change is deployed. Local databases will probably need to be re-generated with `rake db:drop db:setup`.

Reply:
There's a syntax error here. Did you forget to push up a commit?

I got it working with:
```diff
-  config.default_locale = :en
-
   # n.b. we are non-standard using locale names `zhHant` and `zhHans` to avoid
   # hyphens, because of a bug in the plans module. Details were added to the
   # README.
-  config. = [:en, :zhHant, :ko, :zhHans, :so, :es, :tl, :vi, :fi, :sv]
+  config.available_locales = [:en, :zhHant, :ko, :zhHans, :so, :es, :tl, :vi, :fi, :sv]
```

I picked up that `- config.default_locale = :en` bit from https://stackoverflow.com/a/32778473/587955 and it seemed to be necessary although I don't really understand why.

Reply:
Oops, I must have deleted the that when I was writing up my comment. It's fixed and everything should work fine now! 👍